compute/cloud-client/src/test/java/compute/SnippetsIT.java

```diff
@@ -230,4 +266,25 @@ public void testPatchFirewallRule() throws IOException, InterruptedException {
     }
   }
 
+  @Test
+  public void testInstanceOperations()
+      throws IOException, ExecutionException, InterruptedException {
+    Assert.assertSame(getInstanceStatus(MACHINE_NAME), Status.RUNNING);
+    StopInstance.stopInstance(PROJECT_ID, ZONE, MACHINE_NAME);
+    Assert.assertSame(getInstanceStatus(MACHINE_NAME), Status.TERMINATED);
+    StartInstance.startInstance(PROJECT_ID, ZONE, MACHINE_NAME);
+    Assert.assertSame(getInstanceStatus(MACHINE_NAME), Status.RUNNING);
+  }
```

Comment:
These operations might need a bit of time to occur.

Reply:
Added manual wait operation with a timeout.